scripts/linters/pylint_extensions.py

```diff
@@ -2139,6 +2141,54 @@ def visit_classdef(self, node):
             self.add_message('no-dunder-metaclass', node=node)
 
 
+class StringConcatenationChecker(checkers.BaseChecker):
+    """Custom pylint checker which prohibits use of string concatenation and
+     promotes string interpolation instead.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'no-string-concatenation'
+    priority = -1
+    msgs = {
+        'C0035': (
+            'at %s, avoid using string concatenation and replace it with '
+            'string '
+            'interpolation instead.',
+            'no-string-concatenation',
+            'Enforce use of string interpolation over string concatenation.',
+        ),
+    }
+
+    def is_node_string_constant(self, node):
+        """Helper function for visit_binop. Called to check if any of the
+        operand is a string constant.
+        """
+        if (isinstance(node, astroid.nodes.Const) and
+                ('str' in node.pytype() or
+                 'unicode' in node.pytype())):
+            return True
+        return False
```

Comment:
```suggestion
        return (
          isinstance(node, astroid.nodes.Const) and
                ('str' in node.pytype() or 'unicode' in node.pytype()))
```

Reply:
@tanishq67 Is this comment resolved?

Reply:
@tanishq67 Is this comment resolved?

Reply:
Yes, I have taken your suggestions Thanks!